content/kubermatic/master/requirements/cloud_provider/_vsphere.en.md

```diff
@@ -193,3 +193,40 @@ Upstream Kubernetes has been working on the issue for a long time now and tracki
 * <https://github.com/kubernetes/kubernetes/issues/67900>
 * <https://github.com/kubernetes/kubernetes/issues/71829>
 * <https://github.com/kubernetes/kubernetes/issues/75342>
+
+### Datastores and Datastore Clusters
+
+*Datastore* in VMWare vSphere is an abstraction for storage.
+*Datastore Cluster* is a collection of datastores with shared resources and a
+shared management interface.
+
+In KKP *Datastores* are used for two purposes:
+- Storing the VMs files for the worker nodes of vSphere user clusters.
+- Generating the vSphere cloud provider storage configuration for user clusters.
+    In particular to provide the `dafault-datastore` value, that is the default
+    datastore for dynamic volume provisioning.
+
+*Datastore Clusters* can only be used for the first purpose as it cannot be
+specified directly in [vSphere cloud configuration][vsphere-cloud-config].
+
+There are two places where Datastores and Datastore Clusters can be configured
+in KKP
+
+- At datacenter level (either with [Seed CRD]({{< ref "../../concepts/seeds" >}})
+    or [datacenters.yaml]({{< ref "../../concepts/seeds" >}})) is possible to
```

Comment:
Isn't the datacenter.yaml already depricated?

Reply:
yes, it is but 2.15 still supports it and the majority of our customers is still using it

Reply:
ok, if its still supported  then we need to mention it

Reply:
BTW, the link for `datacenters.yaml` is wrong on this line. I will fix it